game/scripts/npc/items/item_phylactery.txt

```diff
@@ -60,10 +60,9 @@
       "bonus_mana"                                        "200 300 450 650 900"
       "bonus_all_stats"                                   "7 12 17 22 27"
       "bonus_mana_regen"                                  "1.0 1.25 1.75 2.5 3.5" //OAA
-      "slow"                                              "50"
+      "slow"                                              "50 60 70 80 90" //OAA
       "bonus_spell_damage"                                "150 450 800 1200 1650"
-      "slow_duration"                                     "1.5"
-      "cooldown_tooltip"                                  "6.0 5.5 5.0 4.5 4.0" //OAA
```

Comment:
If you are still convinced that cooldown needs a nerf, restore this line to: ``"cooldown_tooltip"            "6.0" //OAA``